templates/repo/actions/status.tmpl

```diff
@@ -24,3 +25,4 @@
 {{else}}
 	{{svg "octicon-x-circle-fill" $size (printf "text red %s" $className)}}
 {{end}}
+</span>
```

Comment:
Maybe in the future we should add a 5th argument to `svg` with a k/v map of HTML properties. With that, we could save adding such otherwise unnecessary wrappers.

Reply:
Yes.